common/element_handle_options.go

```diff
@@ -219,7 +219,9 @@ func (f *Files) addFile(ctx context.Context, file goja.Value) error {
 		}
 		f.Payload = append(f.Payload, &parsedFile)
 	case reflect.String: // file path
-		f.Payload = append(f.Payload, &File{Path: file.Export().(string)})
+		if v, ok := file.Export().(string); ok {
+			f.Payload = append(f.Payload, &File{Path: v})
+		}
```

Comment:
Wondering if we need to return an error when `ok` is `false`?

I know i'm being pedantic here since the `if` statement is unnecessary here and only here due to the linter.

Happy for you to ignore this comment.

Reply:
I don't see any chance that the type assertion would fail if the file.exportType() is reflect.String, so I think it would be an unreachable, untestable code.